errtracker/errtracker.go

```diff
@@ -127,6 +128,15 @@ func ReportRepair(repair, errMsg, dupSig string, extra map[string]string) (strin
 	return report(errMsg, dupSig, extra)
 }
 
+func detectVirt() string {
+	cmd := exec.Command("systemd-detect-virt")
```

Comment:
Is this available on Ubuntu 14.04?

Reply:
It is and it seems to work (at lest it detected kvm)